lib/src/characters_impl.dart

```diff
@@ -116,14 +116,16 @@ class StringCharacters extends Iterable<String> implements Characters {
   }
 
   @override
-  bool contains(Object? element) {
-    if (element is String) {
-      if (element.isEmpty) return false;
-      var next =
-          Breaks(element, 0, element.length, stateSoTNoBreak).nextBreak();
-      if (next != element.length) return false;
-      // [element] is single grapheme cluster.
-      return _indexOf(string, element, 0, string.length) >= 0;
+  // ignore: avoid_renaming_method_parameters
+  bool contains(Object? singleCharacterString) {
+    if (singleCharacterString is String) {
```

Comment:
[optional] I think this would read easier without nesting.

```dart
    if (singleCharacterString is! String) return false;
    if (singleCharacterString.isEmpty) return false;
    var next = Breaks(singleCharacterString, 0, singleCharacterString.length,
            stateSoTNoBreak)
        .nextBreak();
```

Reply:
@lrhn - I went ahead with this change. Is this ready to merge?

Reply:
I'm good with it. If you think it's OK to not do a major version increment even if technically breaking, then so am I.